extension/test/e2e/klarna.spec.js

```diff
@@ -70,7 +71,7 @@ describe('::klarnaPayment::', () => {
   it(
     'when payment method is klarna and process is done correctly, ' +
       'then it should successfully finish the payment',
-    async function () {
+    async function func() {
```

Comment:
```suggestion
    async () => {
```

Reply:
Hey Jude, in this method I cannot change it back to lambda function as I have `this.timeout(60000)` inside this method (one line below). Additionally I cannot have anonymous function as eslint is complaining there. Thus it must be a function and also has a name `func`.